problems/1.intro.js

```diff
@@ -4,7 +4,7 @@
  * @returns undefined
  */
 function helloWorld() {
-
+  return undefined
```

Comment:
This function should return "Hello World" as opposed to undefined.

Reply:
Thank you! For helping! I really appreciate it!